eight333.go

```diff
@@ -71,6 +72,29 @@ func (w *SPVWallet) onMerkleBlock(p *peer.Peer, m *wire.MsgMerkleBlock) {
 	if len(w.blockQueue) == 0 && w.blockchain.ChainState() == SYNCING {
 		go w.startChainDownload(p)
 	}
+	if w.blockchain.ChainState() == WAITING {
+		txns, err := w.txstore.Txns().GetAll()
+		if err != nil {
+			log.Error(err)
+			return
+		}
+		now := time.Now()
+		for _, t := range txns {
+			if now.After(t.Timestamp.Add(time.Hour*24*7)) && t.Height == int32(0) {
```

Comment:
Having this time hardcoded will make it very hard to add proper tests for this functionality I think.